codalab/model/bundle_model.py

```diff
@@ -97,6 +97,12 @@ def __init__(self, engine, default_user_info):
         self.public_group_uuid = ''
         self.create_tables()
 
+    def encode_str(self, value):
+        raise NotImplementedError
+
+    def decode_str(self, value):
+        raise NotImplementedError
+
```

Comment:
To make it clear that the concrete classes (i.e. MySQLModel) needs to implement these methods.